.github/workflows/e2e-tests.yml

```
@@ -16,33 +16,40 @@ permissions: {}
 
 jobs:
     e2e-tests:
-        name: 'E2E Tests (${{ matrix.wp }}) (${{ matrix.shard }})'
-        runs-on: ubuntu-latest
+        name: 'E2E Tests (${{ matrix.wp }}) (${{ matrix.project }}) (${{ matrix.shard }})'
+        # Temporarily use ARM macOS for WebKit tests to work around https://github.com/swissspidy/media-experiments/issues/611
+        runs-on: ${{ matrix.project == 'webkit' && 'macos-latest' || 'ubuntu-latest' }}
         continue-on-error: ${{ matrix.experimental == true }}
         strategy:
             fail-fast: false
             matrix:
                 wp: ['latest']
-                shard: ['1/4', '2/4', '3/4', '4/4']
+                project: ['chromium', 'firefox', 'webkit']
+                shard: ['1/2', '2/2']
                 experimental: [false]
                 include:
                     - wp: 'trunk'
-                      shard: '1/4'
+                      shard: '1/2'
+                      project: 'chromium'
                       experimental: true
                     - wp: 'trunk'
-                      shard: '2/4'
-                      experimental: true
-                    - wp: 'trunk'
-                      shard: '3/4'
-                      experimental: true
-                    - wp: 'trunk'
-                      shard: '4/4'
+                      shard: '2/2'
+                      project: 'chromium'
                       experimental: true
 
         steps:
             - name: Checkout
               uses: actions/checkout@v4
 
+            - name: Install Docker (missing on macOS)
+              if: runner.os == 'macOS'
+              run: |
+                brew install docker colima
```

It looks like we may also need to install `docker-compose`.
```suggestion
                brew install docker docker-compose colima
```

---

Just came to the same conclusion :) Let's see how it goes 🤞